src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -10,92 +10,117 @@ import CheckboxWithLabel from '../../components/CheckboxWithLabel';
-                text={translate('common.saveAndContinue')}
-            />
-        </FixedFooter>
-    </>
-);
+                <ScrollView style={[styles.flex1, styles.w100]}>
+                    <View style={[styles.p4]}>
+                        <IdentityForm
+                            onFieldChange={(field, value) => this.setState({[field]: value})}
+                            values={{
+                                firstName: this.state.firstName,
+                                lastName: this.state.lastName,
+                                street: this.state.street,
+                                city: this.state.city,
+                                state: this.state.state,
+                                zipCode: this.state.zipCode,
+                                dob: this.state.dob,
+                                ssnLast4: this.state.ssnLast4,
+                            }}
+                        />
+                        <CheckboxWithLabel
+                            isChecked={false}
+                            onPress={() => {}}
+                            LabelComponent={() => (
+                                <View style={[styles.flex1, styles.pr1]}>
+                                    <Text>
+                                        {this.props.translate('requestorStep.isAuthorized')}
+                                    </Text>
+                                </View>
+                            )}
+                            style={[styles.mt4]}
+                        />
+                        <Text style={[styles.textMicroSupporting, styles.mt5]}>
+                            {this.props.translate('requestorStep.financialRegulations')}
+                            <TextLink
+                                size="micro"
+                                // eslint-disable-next-line max-len
+                                href="https://community.expensify.com/discussion/6983/faq-why-do-i-need-to-provide-personal-documentation-when-setting-up-updating-my-bank-account"
+                            >
+                                {`${this.props.translate('requestorStep.learnMore')}`}
+                            </TextLink>
+                            {' | '}
+                            <TextLink
+                                size="micro"
+                                // eslint-disable-next-line max-len
+                                href="https://community.expensify.com/discussion/5677/deep-dive-security-how-expensify-protects-your-information"
+                            >
+                                {`${this.props.translate('requestorStep.isMyDataSafe')}`}
+                            </TextLink>
+                        </Text>
+                        <Text style={[styles.mt3, styles.textMicroSupporting]}>
+                            {this.props.translate('requestorStep.onFidoConditions')}
+                            <TextLink
+                                size="micro"
```

Comment:
Why not just pass `styles.textMicroSupporting` in the `style` prop instead of introducing a new propType?

Reply:
If you move `styles.link` to the end of the array of styles in `TextLink.js` then you can just pass `                             style={styles.textMicroSupporting}` here

Reply:
Couple reasons, but I did have a similar idea initially...

- the `additionalStyles` are added after everything else so the color in `textMicroSupporting` would overwrite the link styles (which is undesirable since they lose the blue color and appearance as links)

https://github.com/Expensify/Expensify.cash/blob/be71feb42287a5753b2755a7975d4679baefdbc8/src/components/TextLink.js#L34

- since this needed to be added a handful of things it seemed like a good idea to just bake it into the component

Reply:
That said, we could undo this and add it with a style like `textMicroSupportingLink` or remove the "color" from `textMicroSupporting` and add it in with another style. But not really sure what is best here.

Reply:
>If you move styles.link to the end of the array of styles in TextLink.js then you can just pass style={styles.textMicroSupporting} here

Right that just undoes what we did [here](https://github.com/Expensify/Expensify.cash/commit/92e05f4744d04362640d35595fee5f7b4c30668f#diff-646bc9af1fe3161277eeeea6a78ec9ec30245126299b086c109a2ad2ab3863b6R34), but I think the change @roryabraham made makes sense. Styles passed in as props should overwrite some base style and not come before the base styles.

Reply:
one sec, I think I've got a solution